routes.go

```diff
@@ -97,16 +97,83 @@ func upload(c *gin.Context) {
 	}
 	AddLog(1, "uid: "+plug.Owner+"uploaded plug s3id"+plug.S3ID)
 	c.Data(http.StatusOK, "text/html", []byte(`
-	<html>
-	<body>
-		<h2>Uploaded a Plug!</h2>
-		<p>Take a look at what you uploaded! (This does not count towards the views for your Plug!)</p>
-		<p>Due to High Volumes your Plug MUST be approved by a member of any of the following groups (drink, eboard, rtp)! Check /admin</p>
-		<div>
-			<img src="`+S3PresignPlug(plug).String()+`"></img>
-		</div>
-	</body>
-	</html>
+<html>
```

Comment:
Why is this still embedded in a Go executable? 😬

Reply:
Agreed. I’ll take some time later tonight to write up a more complete review, but I think that this change should include the full switch to templates as illustrated in `view_plugs.tmpl`.